web/elm/src/Dashboard.elm

```diff
@@ -405,7 +405,9 @@ update msg model =
                             model
             in
             ( newModel
-            , [ FocusSearchInput, ModifyUrl (NewTopBar.queryStringFromSearch query) ]
+            , [ FocusSearchInput
+              , ModifyUrl (Routes.Dashboard (Just query) |> Routes.toString)
```

Comment:
Ewwwwww, |> in parens. 🤢 

`ModifyUrl (Routes.toString (Routes.Dashboard (Just query)))` or `Just query |> Routes.Dashboard |> Routes.toString |> ModifyUrl`, please.

(Or maybe make ModifyUrl take a route directly, and tostring it in runEffects?)